perses/charts/Chart.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: perses-plugin
```

The suffix `-plugin` is not required.
```suggestion
name: perses
```